src/scripts/content/asana.js

```diff
@@ -70,18 +70,17 @@ togglbutton.render('.SpreadsheetRow .SpreadsheetTaskName:not(.toggl)', { observe
   }
 );
 
-// 2019 My Tasks view, possibly other similar views.
+// 2020 My Tasks view, possibly other similar views.
 togglbutton.render('.MyTasksTaskRow:not(.toggl)', { observe: true },
   function (elem) {
     if (elem.querySelector('.toggl-button')) {
       // Due to the way this UI is rendered, we must check for existence of old buttons manually.
       return;
     }
-    const container = elem.querySelector('.ItemRowTwoColumnStructure-left');
     const descriptionSelector = () => elem.querySelector('.TaskName textarea').textContent;
 
     const projectSelector = () => {
-      const projectElement = elem.querySelector('.TaskRow-pots .Pill');
+      const projectElement = elem.querySelector('.PotPillsContainer');
```

Comment:
This selector doesn't quite work, it will also pick up any tags that are present (breaking the project matching) 😞 since they're all shown as a pill in the same container

<img width="670" alt="Screenshot 2020-06-23 at 11 06 50" src="https://user-images.githubusercontent.com/6432028/85391838-9cb81480-b542-11ea-9c06-85d4cd575852.png">

actually, I think I've worked on the same bug in the other Asana PR here 🙈  sorry I didn't realise it was an existing issue, just happened to notice it https://github.com/toggl/toggl-button/pull/1762/files#diff-50b20c2c72dfe8169e6d571835144c22R91 

do you want to bring that selector ^ in here (if you test that it works)? since this PR still has some other fixes in it we may as well do it independently of the bigger PR that adds tagging support.

Reply:
Damn :see_no_evil: 

That selector doesn't match anything for me at all, I wonder if there are multiple Asana views out again? :thinking: 

Here's what it looks like on mine - a red project followed by a red tag - they're identical :sob: 
![image](https://user-images.githubusercontent.com/50156618/85506796-635fd180-b634-11ea-9353-b5fd8b66ad72.png)

Only thing I can think of is to assume the first pill is a project, and the rest are tags?

Reply:
^ I think that may be the best course here; will need to test all views with it though

Reply:
Updated!

I think the My Tasks page is the only one with this problem, the other views do show tags differently to the project.